linux/src/threadpool_linux.c

```diff
@@ -8,6 +8,14 @@
 #include <semaphore.h>
 #include <time.h>
 #include <string.h>
+#include <signal.h>
+#include <errno.h>
+
+#include <bits/types/__sigval_t.h>         // for __sigval_t
+#include <bits/types/sigevent_t.h>         // for sigevent, sigev_notify_fun...
+#include <bits/types/sigval_t.h>           // for sigval_t
+#include <bits/types/struct_itimerspec.h>  // for itimerspec
+#include <bits/types/timer_t.h>            // for timer_t
```

Comment:
please reconsider these bits/types headers which i suspect are and all APis should be accessed through documented headers.